### DISCUSSION TITLE
Does FDS have a database？

### DISCUSSION THREAD

**Discussion by chenlinjie1213** [firemodels/fds#11682] [General]
Dear Friends, I would like to ask if you have some databases related to FDS? I would like to carry out some machine learning training based on the data from FDS simulation results, thanks.

Thank you.

**Comment by Er9y714** [comment]
This is not an _issue_. You should post this to [discussions](https://github.com/firemodels/fds/discussions).

**Comment by chenlinjie1213** [comment]
ok,Thanks.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "firemodels/fds"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月13日(星期四) 下午4:26
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [firemodels/fds] Does FDS have a database？ (Issue #11679)





 
This is not an issue. You should post this to discussions.
